wordpress-mobile/WordPress-FluxC-Android
# Bump AGP to `8.5.1`

### Description

This PR bumps AGP to 8.5.1 to keep composite build setup working.


### To test

The R8 configuration shouldn't change: `consumerProguardFiles` remains unchanged, and there's no point to obfuscate this open-source SDK before publishing, as the consumers should take care of obfuscating anyway.

Please smoke test the app (either Woo or WordPress/Jetpack) using artifact prepared from this PR.

### Dependencies
- https://github.com/wordpress-mobile/WordPress-Android/pull/21082
- https://github.com/woocommerce/woocommerce-android/pull/12042
- https://github.com/WordPress/gutenberg/pull/64044
